src/modules/markets/markets.controller.ts

```diff
@@ -0,0 +1,29 @@
+import { Controller, Get, Version, CacheInterceptor, UseInterceptors, Query } from '@nestjs/common';
+import { ApiBearerAuth, ApiParam, ApiResponse, ApiTags } from '@nestjs/swagger';
+import { MarketsService } from './markets.service';
+import { MarketEntity } from './entities/market.entity';
+import { CacheTTL } from '@nestjs/common/cache';
```

Comment:
`CacheTTL` doesn't seem to be used. do we need it?

Reply:
removed, was left after testing.